src/PowerShellEditorServices/Session/PSReadLinePromptContext.cs

```diff
@@ -30,7 +30,11 @@ internal class PSReadLinePromptContext : IPromptContext {
             [System.Diagnostics.DebuggerStepThrough()]
             param()
             end {
-                $module = Get-Module -ListAvailable PSReadLine | Where-Object Version -ge '2.0.0' | Sort-Object -Descending Version | Select-Object -First 1
+                $module = Get-Module -ListAvailable PSReadLine `
+                    | Where-Object Version -ge '2.0.0' `
+                    | Where-Object { $psd = Import-PowerShellDataFile $_.Path; $release = $psd.PrivateData.PSData.Prerelease; -not (@('beta1', 'beta2') -contains $release) } `
```

Comment:
Maybe simplify a bit with `$release -notin @('beta1','beta2')`?  If only we had a `SemanticVersion` class we could use for sorting in Win PS 5.

Reply:
Aha! I wondered if an operator like that existed